src/popup/pages/TransactionDetails.vue

```diff
@@ -85,6 +85,64 @@
             </template>
           </DetailsItem>
 
+          <DetailsItem
+            v-if="multisigTransactionFeePaidBy"
+            :label="$t('pages.transactionDetails.feePaidBy')"
+            small
+          >
+            <template #value>
```

Comment:
I changed the `<DetailsItem>` component that we don't need to use the `#value` slot as now it's the default one. So in most cases it's:
```
<DetailsItem label="...">
  Stuff goes here
</DetailsItem>
```